client/src/components/Charts/BarChart.js

```diff
@@ -1,63 +1,69 @@
 import React from 'react';
 import propTypes from 'prop-types';
 import { Bar } from 'react-chartjs-2';
+import { useTranslation } from 'react-i18next';
 
-const BarChart = ({ centers, frequencies }) => (
-  <div className="p-4 m-auto md:w-2/3 lg:w-1/2">
-    <Bar
-      data={{
-        labels: centers,
-        datasets: [
-          {
-            label: 'Frequency',
-            data: frequencies,
-            backgroundColor: 'green',
-            barThickness: 'flex',
-            barPercentage: 1,
-          },
-        ],
-      }}
-      options={{
-        title: {
-          display: true,
-          text: 'Custom Chart Title',
-        },
-        responsive: true,
-        maintainAspectRatio: true,
-        aspectRatio: 1,
-        plugins: {
-          datalabels: {
-            font: {
-              size: 0,
-            },
-          },
-        },
-        scales: {
-          xAxes: [
+const BarChart = ({ centers, frequencies, colors }) => {
+  const { t } = useTranslation();
+  return (
+    <div className="p-4 m-auto h-half-screen lg:w-2/3">
+      <Bar
```

Comment:
**split data and options like this:**  
`<Bar data={data} options={options} />`